packages/ses-ava/package.json

```diff
@@ -36,12 +37,12 @@
     "test": "ava"
   },
   "dependencies": {
+    "@endo/env-options": "^1.1.1",
+    "@endo/init": "^1.0.4",
+    "ava": "^5.3.0",
```

Comment:
I have no experience with peer dependencies and don't understand them. Are you ok with this peer dependencies change not happening in this PR, given that placing ses-ava in devDependencies solved the immediate problem?

Any such deeper change in our dependency structure for endo I'd like to leave in @kriskowal 's hands anyway. @kriskowal ?

Reply:
> placing ses-ava in devDependencies solved the immediate problem

I expect there will be other problems from using `dependencies`. I'm willing to defer to @kriskowal .

Reply:
I prefer adding `ava` to the `dependencies` of `@endo/ses-ava` over making it a peer dependency.

Using `peerDependencies` means that a application package has to retain a `devDependencies` entry for both `@endo/ses-ava` and a compatible version range of `ava`. I like `peerDependencies` for packages that might or might not be retained by the application package depending on whether they’re actually used, but in this case, Ava is always necessary for the functioning of `@endo/ses-ava`.

Please note that by “application”, I mean the package that ultimately retains the `devDependencies` tree.

Reply:
Thanks. Resolving.

Reply:
One of my concerns was version coupling and duplicate deps. I agree that on balance `dependencies` is the better choice.

But to reduce the coupling, mind making the version range accept Ava 6 too? I don't see any incompatible changes in https://github.com/avajs/ava/releases/tag/v6.0.0 (though you would know better) And in the other direction, I don't know that it necessarily requires 5.3 vs 5.0.

Reply:
@kriskowal just noting:

While looking into this, I notice that at 
https://github.com/endojs/endo/blob/5186a5040689f27719f79f1f8f808dbc46b7af0d/packages/init/package.json#L34

@endo/init currently depends on Ava "^5.1.1" whereas all other endo packages currently depend on Ava "^5.3.0". I am proceeding assuming this is a mistake unless I hear otherwise.

What tooling do we have for checking such consistency among package dependencies?

Reply:
`./scripts/all-versions.sh` gives you the most recent of all transitive dependencies and piping that to `./scripts/set-verison.sh` will give you a diff for any discrepancies. Some of those discrepancies are deliberate, e.g., `typescript` is pinned with `~` instead of `^` because of differences in how they interpret their semver obligations.

Reply:
Ava 6.0.0 does not work with Node 16. Endo still seems to require Node 16, or at least endo's CI tests still do:

![image](https://github.com/endojs/endo/assets/273868/0305940a-5737-4b59-97b8-c25b5e459913)

So, reverting the Ava dependencies back to "^5.3.0". If it does not cause CI problems, I will keep the change of @endo/init from Ava "^5.1.1" to "^5.3.0".

Reply:
Done!